lms/djangoapps/teams/tests/test_views.py

```diff
@@ -2794,3 +2794,17 @@ def test_create_membership_via_upload_using_external_key_invalid(self):
             response_text['errors'],
             ['User name/email/external key: a_user_external_key_invalid does not exist.']
         )
+
+    def test_upload_non_ascii(self):
+        csv_content = 'user,mode,topic_0' + '\n'
+        team_name = u'著文企臺個'
```

Comment:
I would be curious what that name means. Also, trivial: we are using a variable called team_name for both user_name and team_name. Rename to `test_name` ?

Reply:
@schenedx ?

Reply:
fair.  This is just the team name from the above "chinese team"